src/networks.js

```diff
@@ -10,15 +11,47 @@ export const RINKEBY_STAGING_COURT =
   '0x52180Af656A1923024D1ACcF1D827AB85cE48878'
 
 export const networks = {
-  rpc: { court: '0xD833215cBcc3f914bD1C9ece3EE7BF8B14f841bb' },
+  local: { court: '0xD833215cBcc3f914bD1C9ece3EE7BF8B14f841bb' },
   ropsten: { court: '0x3b26bc496aebaed5b3E0E81cDE6B582CDe71396e' },
   rinkeby: {
     // Use the 'usability' Court address if declared
```

Comment:
We might need to update this comment now that we're also including the staging environment on rinkeby